go/chat/boxer.go

```diff
@@ -281,6 +281,7 @@ func (b *Boxer) checkInvariants(ctx context.Context, convID chat1.ConversationID
 }
 
 func (b *Boxer) unbox(ctx context.Context, boxed chat1.MessageBoxed, encryptionKey types.CryptKey) (*chat1.MessageUnboxedValid, UnboxingError) {
+	b.Debug(ctx, "Unboxing message version: %v", boxed.Version)
```

Comment:
fyi locally (at least on this branch) when sending a message it unboxes 2, sometimes 3 times, where I would expect 0 or maybe 1.